internal/api/handlers/device_definition_handler.go

```diff
@@ -75,3 +76,37 @@ func GetDeviceDefinitionByMMY(m mediator.Mediator) fiber.Handler {
 		return c.Status(fiber.StatusOK).JSON(result)
 	}
 }
+
+func GetDeviceDefinitionByDynamicFilter(m mediator.Mediator) fiber.Handler {
```

Comment:
do we need a fiber handler since this would be done over grpc? eg. no need for a rest endpoint

Reply:
I think we don't 🤔 ...